definitions/variable/electricity.yaml

```
@@ -28,3 +28,10 @@
 - Discount Rate|Electricity:
     description: Discount rate applied to electricity-related investments
     unit: '%'
+- Final Energy|Electricity [Share]:
+    description: Share of load within the specified region
+    unit: ''
+- Network|Electricity|Capacity:
+     description: Power transmission capacity
+     unit: 'MW'
```

No need for quotes if the value is just a simple string.
```suggestion
     unit: MW
```